packages/react/src/components/table/table.tsx

```diff
@@ -277,25 +335,33 @@ export const Table = <T extends object>({
             <StyledTHead>
                 {table.getHeaderGroups().map((headerGroup) => (
                     <TableHeader<T>
+                        key={headerGroup.id}
                         headerGroup={headerGroup}
                         sticky={stickyHeader}
                     />
                 ))}
             </StyledTHead>
             <StyledTBody>
-                {table.getRowModel().rows.map((row) => (
-                    <TableRow<T>
-                        striped={striped}
-                        error={!!(row.original as CustomRowProps).error}
-                        row={row}
-                        onClick={onRowClick}
-                    />
-                ))}
+                {table.getRowModel().rows.map((row) => {
+                    const rowOriginal = row.original as TableData<T>;
+                    return (
+                        <Fragment key={row.id}>
+                            <TableRow<T>
+                                striped={striped}
+                                error={!!rowOriginal.error}
+                                row={row}
+                                onClick={onRowClick}
+                            />
+                            {rowOriginal.subContent !== undefined && row.getIsExpanded() && rowOriginal.subContent}
```

Comment:
Ce serait pas équivalent vu qu'undefined va évaluer à false?
```suggestion
                            {row.getIsExpanded() && rowOriginal.subContent}
```